tests/test_merge.py

```diff
@@ -215,18 +206,41 @@ def test_merge_conflicts(
 
         r = cli_runner.invoke(["merge", "alternate"])
 
-        assert r.exit_code == NOT_YET_IMPLEMENTED, r
+        assert r.exit_code == INVALID_OPERATION, r
         assert "conflict" in r.stdout
         assert base_commit_id in r.stdout
         assert alternate_commit_id in r.stdout
         assert master_commit_id in r.stdout
 
+        assert "Use an interactive terminal to resolve merge conflicts" in r.stderr
+
+        def feature_name(pk):
+            return f"{data.LAYER}:{data.LAYER_PK}={pk}"
+
         # Only modified in alternate:
-        assert f"{layer}:{sample_pks[0]}" not in r.stdout
+        assert feature_name(sample_pks[0]) not in r.stdout
         # Modified exactly the same in both branches:
-        assert f"{layer}:{sample_pks[1]}" not in r.stdout
+        assert feature_name(sample_pks[1]) not in r.stdout
         # These two are merge conflicts:
-        assert f"{layer}:{sample_pks[2]}" in r.stdout
-        assert f"{layer}:{sample_pks[3]}" in r.stdout
+        assert feature_name(sample_pks[2]) in r.stdout
+        assert feature_name(sample_pks[3]) in r.stdout
         # Only modified in master
-        assert f"{layer}:{sample_pks[3]}" in r.stdout
+        assert feature_name(sample_pks[4]) not in r.stdout
+
+        r = cli_runner.invoke(["merge", "alternate"], input="o\nt\n")
+        assert r.exit_code == 0, r
+        assert repo.head.target.hex != alternate_commit_id
+        assert repo.head.target.hex != master_commit_id
+
+        r = cli_runner.invoke(["diff", master_commit_id])
+        assert r.exit_code == 0, r
+
+        # Diff merged in from alternate automatically
+        assert feature_name(sample_pks[0]) in r.stdout
+        # Diff merged in from alternate explicitly with "t" for theirs
```

Comment:
can we have a test for ours & ancestor resolution;
and a test for types of changes (meta, others?) that don't work atm?